### ISSUE TITLE
"Web Components" should be removed from "Web technology for developers" page

### ISSUE THREAD

**Issue by DanKaplanSES** [mdn/content#35452]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web#web_technology_references

### What information was incorrect, unhelpful, or incomplete?

[This page](https://github.com/mdn/content/edit/main/files/en-us/web/index.md) contains this html:

```
<section id="Quick_links">
  {{ListSubpages("", 1, 0, 1)}}
</section>
```

I can't find this exact HTML in the webpage, but I think it's responsible for building the left-hand sidebar. 

![image](https://github.com/user-attachments/assets/f6517942-0269-42f2-9af5-b7c29757dc8f)

If I'm correct, it doesn't seem to be populating the list consistently. As far as I can tell, it should include [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components), but it's not in that list.

### What did you expect to see?

I expected to see "Web Components" in this sidebar:

![image](https://github.com/user-attachments/assets/f6517942-0269-42f2-9af5-b7c29757dc8f)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/index.md
* Last commit: https://github.com/mdn/content/commit/cb132bc83b660e51be8959de5336c00b08030104
* Document last modified: 2024-08-01T16:24:09.000Z

</details>

**Comment by Josh-Cena** [comment]
It only includes top-level categories. "Web components" is an API category. Why would you expect to see it here?

**Comment by Josh-Cena** [comment]
Ah, I see where you are coming from. "Web Components" is included in the content. It should be removed from this page.

**Comment by DanKaplanSES** [comment]
I'll add context I should've included to the original post. Sorry for the late reply! 

> Ah, I see where you are coming from. "Web Components" is included in the content. It should be removed from this page.

Actually I was expecting it to stay and for it to be added on the left-hand side; I'm a little confused at the level of granularity on this page, but I only realized my confused after reading your reply. For example, this page links to Web APIs, JavaScript, and Events. Events seem like a subsection of the former two, and from my uneducated perspective, events seems like a similar level of granularity to Web Components.

Since to me it felt like Web Components belonged on this page, I searched for it in the left-hand side filter. When I couldn't find it, I was prepared to move on, but then I noticed it was in the body of the article. I then went spelunking into the source code to try to figure out why this was missing from the left-hand side even though it was included in the body. I never really figured that out and I'm still confused about this: I probably misunderstand what `ListSubpages` does on this page.
